Samples/iOS-Swift/iOS-SwiftUITests/SentryFileIOTrackingIntegrationTests.swift

```diff
@@ -0,0 +1,246 @@
+        SentrySDK.start(options: fixture.getOptions())
+        
+        guard let jsonFile = getBigFilePath() else {
+            XCTFail("Could not open Resource")
+            return
+        }
+        
+        assertSpans(1) {
+            let data = try? NSData(contentsOfFile: jsonFile, options: .uncached)
+            XCTAssertEqual(data?.count, 341_431)
+        }
+    }
+    
+    func test_WritingBigFile() {
+        guard let jsonFile = getBigFilePath() else {
+            XCTFail("Could not open Resource")
+            return
+        }
+        
+        guard let data = try? NSData(contentsOfFile: jsonFile, options: .uncached) else {
+            XCTFail("Could not load File")
+            return
+        }
+        
+        SentrySDK.start(options: fixture.getOptions())
+        
+        assertSpans(1) {
+            try? data.write(to: fixture.fileURL, options: .atomic)
+
+            let size = try? fixture.fileURL.resourceValues(forKeys: [.fileSizeKey]).fileSize ?? 0
+            
+            XCTAssertEqual(size, 341_431)
+        }
+    }
+    
+    func getBigFilePath() -> String? {
+        let bundle = Bundle(for: type(of: self))
+        
+        return bundle.path(forResource: "Resources/fatal-error-binary-images-message2", ofType: "json")
+        ?? bundle.path(forResource: "fatal-error-binary-images-message2", ofType: "json")
+    }
+    
+    func test_DataConsistency_readUrl() {
+        SentrySDK.start(options: fixture.getOptions())
+        
+        let randomValue = UUID().uuidString
+        try? randomValue.data(using: .utf8)?.write(to: fixture.fileURL, options: .atomic)
+        print("\(String(describing: fixture.fileURL))")
+        guard let data = try? Data(contentsOf: fixture.fileURL, options: .uncached) else {
+            XCTFail("Could not load resource file")
+            return
+        }
+        let readValue = String(data: data, encoding: .utf8)
+        XCTAssertEqual(randomValue, readValue)
+    }
+    
+    func test_DataConsistency_readPath() {
+        SentrySDK.start(options: fixture.getOptions())
+        
+        let randomValue = UUID().uuidString
+        try? randomValue.data(using: .utf8)?.write(to: fixture.fileURL, options: .atomic)
+        guard let data = try? NSData(contentsOfFile: fixture.filePath) as Data else {
+            XCTFail("Could not load resource file")
+            return
+        }
+        let readValue = String(data: data, encoding: .utf8)
+        XCTAssertEqual(randomValue, readValue)
+    }
+    
+    private func assertSpans(_ spansCount: Int = 0, _ block : () -> Void) {
+        let parentTransaction = SentrySDK.startTransaction(name: "Transaction", operation: "Test", bindToScope: true)
+        
+        block()
+        
+        let childrenSelector = NSSelectorFromString("children")
+        
+        let children = parentTransaction.perform(childrenSelector).takeUnretainedValue() as? [Span]
+        XCTAssertEqual(children?.count, spansCount)
```

Comment:
`m`: Instead of just asserting the number of spans, can we also check if the created span is correct. A simple check for the description or something.

Reply:
Sure, it make sense

Reply:
I added a test to verify the operation. 

Descriptions depends on the file path, size and operation type. We have more tests for description on another unit test.